pyproject.toml

```diff
@@ -99,6 +92,42 @@ bump = true
 enable = true
 style = "pep440"
 
+[tool.ruff]
+target-version = "py38"
+line-length = 127
+
+select = [
+    "F",     # Pyflakes
+    "W",     # Pycodestyle (warnigns)
+    "E",     # Pycodestyle (errors)
+    "N",     # pep8-naming
+    "ANN",   # flake8-annotations
+    "B",     # flake8-bugbear
+    "FA",    # flake8-future-annotations
+    "TID",   # flake8-tidy-imports
+    "UP007", # flake8-new-union-types equivalent rule
+    "RUF",   # ruff's custom rules
+]
+
+ignore = [
```

Comment:
```suggestion
ignore = [
    "E203",   # Incompatible with black in some moments
```

See https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#flake8

Reply:
This is disabled by default in ruff (part of nursery), see:
![image](https://github.com/py-mine/mcstatus/assets/20902250/adf50ae0-f766-4585-811b-5a71da08347f)
![image](https://github.com/py-mine/mcstatus/assets/20902250/05376e73-5cef-4733-a04a-ed5591aa1862)
![image](https://github.com/py-mine/mcstatus/assets/20902250/fb2b3893-444e-4e97-a723-63d11c33270f)